templates/react/package.json

```diff
@@ -27,5 +27,29 @@
   "engines": {
     "node": "^18",
     "pnpm": "^8 || ^9"
+  },
+  "pnpm": {
```

Comment:
this is here for the same reasons as we had before where the wagmi instance used by rainbowkit isn't resolved to the same place as the wagmi instance used by the template/app, so we get issues with wagmi config not found

it means that you don't get live updates when developing the template and to get the latest version of MUD deps, you have to do something like
```
rm -rf node_modules packages/*/node_modules; pnpm install
```
to forcefully remove all deps and reinstall (even `pnpm install --force` isn't enough)

Reply:
I found another approach that works a bit better and has less downsides.